### PR TITLE
Added support page and link and tweaked the about page

### PR DESCRIPTION
**Pull request by vickytnz**
New support page
support page with details of slack channels

About page changed
| Before | After | 
| -- | --  |
| Before about page |  after about page with contribution section changed | 

Footer
| Example | Image |
| -- | --  |
| Before |   footer link - with About and support  NHS prototype kit on GitHub Cookies | 
 | After |    footer link with About  Support  NHS prototype kit on GitHub  Cookies |

### CODE REVIEW

File: app/views/support.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      <ul>
+          <li><a href="mailto:england.service-manual@nhs.net?subject=NHS%20prototype%20kit%20-%20About">email the service manual team</a></li>
+          <li>use Slack - if you have an NHS England email or have been given access to the NHS England slack instance you can use the <a href="https://nhsdigitalcorporate.enterprise.slack.com/archives/C042J3MTJG2">NHS England #prototype-kit slack channel</a>. If you do not you can also use the <a href="https://nhs-service-manual.slack.com/messages/CFYL2GDGW"> NHS digital service manual #prototype-kit Slack channel</a></li>
+          <li>add a bug or view issues on on GitHub for the <a href="https://github.com/nhsuk/nhsuk-prototype-kit/issues">prototype kit</a> or  <a href="https://github.com/nhsuk/nhsuk.service-manual.prototype-kit.docs/issues">this documentation site</a>  </li>
+        </ul>
+      <p>You can also get involved with <a href="/about#contributing">contributing</a>.</p>
+      <p>There is a different place to <a href="https://service-manual.nhs.uk/get-in-touch">get support for the NHS general service manual (including the design system)</a>.</p>
```
Comment (frankieroberto) [inline app/views/support.html:27]: I think we could drop these? If people use the channels for general service manual queries then they can always be redirected later, depending on the channel?

```suggestion
```

File: app/views/support.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    <div class="nhsuk-grid-column-two-thirds">
+
+      <h1 id="support">Support</h1>
+      <p>The NHS prototype kit is maintained by <a href="https://service-manual.nhs.uk/service-manual-team">NHS England service manual team</a> with help from community contributors. If you&#39;ve got a question or need support you can:</p>
+      <ul>
+          <li><a href="mailto:england.service-manual@nhs.net?subject=NHS%20prototype%20kit%20-%20About">email the service manual team</a></li>
+          <li>use Slack - if you have an NHS England email or have been given access to the NHS England slack instance you can use the <a href="https://nhsdigitalcorporate.enterprise.slack.com/archives/C042J3MTJG2">NHS England #prototype-kit slack channel</a>. If you do not you can also use the <a href="https://nhs-service-manual.slack.com/messages/CFYL2GDGW"> NHS digital service manual #prototype-kit Slack channel</a></li>
+          <li>add a bug or view issues on on GitHub for the <a href="https://github.com/nhsuk/nhsuk-prototype-kit/issues">prototype kit</a> or  <a href="https://github.com/nhsuk/nhsuk.service-manual.prototype-kit.docs/issues">this documentation site</a>  </li>
+        </ul>
```
Comment (frankieroberto) [inline app/views/support.html:25]: This bullet list of 3 items containing 5 links feels a bit overwhelming to me. 

I wonder if we could use a pattern more like [Contact a department or team](https://design-system.service.gov.uk/patterns/contact-a-department-or-service-team/) with headings?

Also think for GitHub we can just direct everyone to open issues on the main prototype kit repo here: https://github.com/nhsuk/nhsuk-prototype-kit/issues/new/choose - and if they’re to do with the documentation website then we can just move them across? (Saves people having to think about which is which)

Something like this?

```suggestion
      <h2>Email</h2>
      <p><a href="mailto:england.service-manual@nhs.net?subject=NHS%20prototype%20kit%20-%20About">england.service-manual@nhs.net</a></p>
      
      <h2>Slack</h2>
      
      <p><a href="https://nhsdigitalcorporate.enterprise.slack.com/archives/C042J3MTJG2">#prototype-kit channel</a> on the NHS England Slack.</p>
      
      <p><a href="https://nhs-service-manual.slack.com/messages/CFYL2GDGW">#prototype-kit channel</a> on the NHS digital service manual Slack</p>
      
      <h2>GitHub</h2>
      
      <p><a href="https://github.com/nhsuk/nhsuk-prototype-kit/issues/new/choose">Report a bug or issue</a> - requires a free GitHub account.</p>
```

File: app/views/support.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{% extends 'layout.html' %}
+
+{% block pageTitle %}
+  Cookies - NHS prototype kit
```
Comment (frankieroberto) [inline app/views/support.html:4]:
```suggestion
  Support - NHS prototype kit
```

File: app/views/about.html [hunk 4]
```diff
@@ -22,15 +22,9 @@ <h1>About</h1>
 
       <h2>Support</h2>
 
-      <p>The NHS prototype kit is maintained by NHS England. If you've got a question or need support you can:</p>
+      <p>View our <a href="/support">support</a> section.</p>
```
Comment (frankieroberto) [inline app/views/about.html:25]: Could drop both lines to save the awkwardness of cross-linking?

**Review by frankieroberto — COMMENTED** [review]
Made some suggestions.

**Review by vickytnz — COMMENTED** [review]
All changes make sense

File: app/views/about.html (hunk 4 above)
Comment (vickytnz) [reply app/views/about.html:25]: Yep agreed - going to see if this makes the commit stick.

**Review by vickytnz — COMMENTED** [review]
All changes accepted

**Comment by vickytnz** [comment]
New support page
new support page


About page changed
| Before | After | 
| -- | --  |
|  Before about page |  changed about page with link to support  | 


Footer
| Example | Image |
| -- | --  |
| Before |   footer link - with About and support  NHS prototype kit on GitHub Cookies | 
 | After |    footer link with About  Support  NHS prototype kit on GitHub  Cookies |

**Comment by frankieroberto** [comment]
@vickytnz just realised the 2 slack links need different link text for accessibility reasons. Any ideas?

**Comment by vickytnz** [comment]
> @vickytnz just realised the 2 slack links need different link text for accessibility reasons. Any ideas?

@frankieroberto Ah yeah that's why in the original one I led with the Slack instance as is does make a difference for some people which one to use. I actually I think on the second we can get rid of the NHS service manual option.

* NHS England slack #prototype-kit channel
* service manual #prototype-kit channel 

We are supposed to follow the 'if ... then' format so if we need anything more than the header at the start then we probably need to roll back to explaining who can use which instance

**Comment by frankieroberto** [comment]
@vickytnz have tweaked slightly. How’s this:

support-page

**Review by frankieroberto — APPROVED** [review]

**Comment by vickytnz** [comment]
Final couple of tweaks that i've picked up after being away from it :

- made first paragraph lede size like other sections
- flipped the language of the team to be 'service manual team at NHS England' to be consistent with the design system

support page with lede section of contact to team and then set of options

**Comment by frankieroberto** [comment]
@vickytnz nice! Think we’re good to merge. 🚢

**Pull request merged**